lib/apiservers/engine/backends/volume.go

```diff
@@ -234,7 +242,7 @@ func translateInputsToPortlayerRequestModel(name, driverName string, opts, label
 	model.Metadata = make(map[string]string)
 	model.Metadata[dockerMetadataModelKey] = metadata
 	if err := validateDriverArgs(opts, model); err != nil {
-		return nil, err
+		return nil, fmt.Errorf("Bad Driver Arg: %s", err)
```

Comment:
@hickeng I now see what your quetion was. No it is not. But we check for the driver sooner than this portion. There is an integration test that checks this. Though I can probably also add it to the unit tests that are there.